docs/compose/.env

```diff
@@ -97,6 +97,9 @@ COMPRESSION_LEVEL=
 # Web settings
 ###################################
 
+# Module to forward to from /
+WEB_DEFAULT=webmail
```

Comment:
There is not any other module to choose from. If any other string is set here, the forward will not happen anymore. For instance, setting this to `admin` will not have the advertised effect and a 404 will be served.

Perhaps it's better to make this a `boolean` type and call it something like `WEBMAIL_REDIRECT`.

Reply:
Under this scheme it may be feasible to point to admin (which this change doesn't support yet, I admit) or future additions if mailu's scope expands.

Making this a boolean means that we're locking ourselves in this corner: what would happen if, in some future version, both WEBMAIL_REDIRECT and ADMIN_REDIRECT (or CALENDAR_REDIRECT or whatever else may come up) are set to true?

If you prefer, I'll change it to the boolean scheme.

Reply:
Personally, I'm not a fan of redirects. See issue #662 ;). I was looking into removing it altogether. However, making it a user's choice is a much better option.

The reason why I suggested a `boolean` type, is that the current proposal seems a bit messy.
- We have `WEB_WEBMAIL=/webmail` with slash
- We have `WEB_DEFAULT=webmail` without slash

Now, a novice user comes along and changed the path to the webmail service. I _might_ think to change `WEB_DEFAULT` to the the same string. He does not know that the string in this variable need to match exactly to a module name.

So there are 3 real options worth considering:

1. Keep your approach and document clearly what are the valid module names. Create `if` statements for the other modules. (Personally, I think this is a dirty approach)
2. Lock it down in a boolean. Simple and straightforward.
3. (New idea) Call it `WEBROOT_REDIRECT` and redirect literally to this string. If its empty, don't redirect.  This way users can customize however they wish. They can even redirect to an external site. It creates flexibility if we will ever implement the support of [multiple webmails](https://github.com/Mailu/Mailu/issues/343#issuecomment-430784490).

Reply:
I like the WEBROOT_REDIRECT option and implemented it. In the spirit of "one change per commit" I kept the default behavior the same, ie. it redirects to /webmail by default (since the default behavior also includes providing webmail).